src/Entrust/Traits/EntrustUserTrait.php

```diff
@@ -193,24 +166,25 @@ public function ability($roles, $permissions, $options = [])
             $checkedPermissions[$permission] = $this->can($permission);
         }
 
+        $validateAll = false;
```

Comment:
setting the $validateAll = false; as a default here and override it only if the condition was matched.
(instead of putting it in the else block.)